discord/abc.py

```diff
@@ -1335,6 +1345,9 @@ async def send(
                 raise InvalidArgument('embeds parameter must be a list of up to 10 elements')
             embeds = [embed.to_dict() for embed in embeds]
 
+        if stickers:
```

Comment:
```suggestion
        if stickers is not None:
```

To stay consistent with the rest of the library, I think? At least, the rest of this method.